pyproject.toml

```diff
@@ -20,6 +75,20 @@ exclude = '''
 [tool.isort]
 profile = "black"
 line_length = 100
-py_version=310
+py_version = 310
 length_sort_sections = ["stdlib", "thirdparty", "firstparty", "localfolder"]
 lines_between_sections = 1
+
+[tool.pytest.ini_options]
+minversion = "3.5"
```

Comment:
I'm fairly sure that you're using `pytest` features newer than v3.5. Maybe bump this up to v7 or v8?